lib/cadet/updater/CS1101S.ex

```diff
@@ -0,0 +1,69 @@
+defmodule Cadet.Updater.CS1101S do
+  @moduledoc """
+  Pulls content from the remote cs1101s repository into a local one.
+  """
+
+  @remote_repo (if Mix.env() != :test do
+                  Dotenv.load().values["01S_REPOSITORY"]
+                else
+                  "test/remote_repo"
+                end)
+  @local_name if Mix.env() != :test, do: "cs1101s", else: "test/local_repo"
+  @key_file Dotenv.load().values["01S_RSA_KEY"]
+
+  require Logger
+
+  def clone() do
+    Logger.info("Cloning CS1101S: Started")
+    git("clone", [@remote_repo, @local_name])
+    Logger.info("Cloning CS1101S: Done")
+  end
+
+  def update() do
+    Logger.info("Updating CS1101S...")
+    git("fetch")
```

Comment:
Why do you need to `git fetch`? Afaik `git pull` = `git fetch` + `git merge`

Reply:
You're 100% right here I just like to fetch before I pull so often that I forgot pull does an implicit fetch.Fixed with 69da032.